### ISSUE TITLE
Egress pods on crashloopbackoff with `open /sys/fs/cgroup/cpu.stat: no such file or directory` on EKS

### ISSUE THREAD

**Issue by bmbferreira** [livekit/egress#150]
The change done in https://github.com/livekit/egress/pull/144 is causing the pod's to crash on startup with the following log message:
```
+ export TMPDIR=/tmp/lkegress
+ TMPDIR=/tmp/lkegress
+ rm -rf /var/run/pulse /var/lib/pulse /root/.config/pulse
+ pulseaudio -D --verbose --exit-idle-time=-1 --system --disallow-exit
W: [pulseaudio] main.c: Running in system mode, but --disallow-module-loading not set.
N: [pulseaudio] main.c: Running in system mode, forcibly disabling SHM mode.
I: [pulseaudio] main.c: Daemon startup successful.
+ '[' -z /tmp/lkegress ']'
+ mkdir -p /tmp/lkegress
+ exec egress
{"level":"info","ts":1666453820.91946,"logger":"egress","caller":"redis/redis.go:53","msg":"connecting to redis","nodeID":"NE_nopW5aQoMiJ4","sentinel":false,"addr":"lr.abcdef.ng.0001.euw1.cache.amazonaws.com:6379"}
open /sys/fs/cgroup/cpu.stat: no such file or directory
```

I'm running egress on AWS EKS, with `c7g.4xlarge` instances and [Bottlerocket AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami-bottlerocket.html). 

I forked the repo, reverted [this PR](https://github.com/livekit/egress/pull/144), tested it and can confirm that it works without that change.

**Comment by nitedani** [comment]
Crashes in docker with the same error.

**Comment by bmbferreira** [comment]
> Crashes in docker with the same error.

yes, maybe I was not explicit, the error I'm having is inside a docker container as well, running on EKS, using bottlerocket AMI. Didn't try with a different AMI though, so I don't know if it's related.

**Comment by duiniuluantanqin** [comment]
the same error with me. And I find there is no  file `cpu.stat` in directory `sys/fs/cgroup/`,  on the host machine.
But I have not solution by now.

**Comment by ozgur-d** [comment]
update ubuntu to 20.10 or use 22.04. it will be fixed.

**Comment by duiniuluantanqin** [comment]
> 

you are right, but my system is centos.

**Comment by meakbiyik** [comment]
Same issue here, had to revert to v1.3.0 -upgrading to 22.04 did not help on WSL.

**Comment by davidzhao** [comment]
this was a breakage in v1.5.0. we'll be releasing v1.5.1 tomorrow with a fix for this.

**Comment by davidzhao** [comment]
Fixed in https://github.com/livekit/egress/releases/tag/v1.5.1

**Comment by meakbiyik** [comment]
@davidzhao it sadly does not seem to have resolved on WSL 2, I have a crashing deployment with the error `failed reading cpu stats file`.

**Comment by meakbiyik** [comment]
The new commit seems to be the solution for this issue 🥇 When is the planned release?

**Comment by biglittlebigben** [comment]
This should be fixed in v1.5.2
